### ISSUE TITLE
Concernings about the ChIP-Seq tutorial 'Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1)'

### ISSUE THREAD

**Issue by joachimwolff** [galaxyproject/training-material#456]
We have taught the tutorial about ChIP-seq: [Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1)](https://galaxyproject.github.io/training-material//topics/chip-seq/tutorials/tal1-binding-site-identification/tutorial.html) as it is published. During the preparation of the session some questions appeared that we can not answer with the given material. 

1. If you compare the file sizes of all G1E-cell types and all megakaryocytes cells you see that all G1E samples have a file size between 30 - 60 MB, except of G1E_input_R2 which is having around 400 MB. For megakaryocytes it looks the same: all samples have a file size of around 400 MB except of megakaryocytes_input_R1 which is having a file size of around 60 MB. Are these two files swapped around by mistake?

2. Step 5: plotFingerprint: 
a ) G1E_input_R1 and G1E-TAl1_R1 are having the same 'enrichment'. (blue and green lines). We think this is not a good example for teaching. The control and the treatment should differ more.
![image](https://user-images.githubusercontent.com/13435136/28916678-5153d7b6-7843-11e7-8a67-8a57c3389c17.png)
b) The megakaryoctyes do not show a distinction between control and samples.
![image](https://user-images.githubusercontent.com/13435136/28916791-b47db212-7843-11e7-8c3c-1177165392eb.png)

3. Compute Matrix and Heatmap plotting
a) For Step 10: ComputeMatrix for heatmap plotting:
Step 10.3. "Under “Score file”, select the bigWigs (log2 ratios from bamCompare)." indicates not exactly which files (samples) we should select for 'Score file'. Given the image below we assumed to choose all samples for the score file i.e. the results of bamCompare for `G1E_input_R1+G1E_TAL1_R1`;  `G1E_input_R2+G1E_TAL1_R2`;  `mega_input_R1+mega_TAL1_R1`;  `mega_input_R1+mega_TAL1_R1`;. But for us it does not make sense to use one region for one cell type and to use the score files for all cell types. We think it would make more sense to use the same cell types for the region and score file.
b) Assuming the setup is correct, we would expect that for the region file for cell type G1E that the enrichment would be seen for this cell type too. But in the given heatmap plot there is no enrichment for G1E compared to the megakaryocytes. 

4. Assuming everything in the tutorial is correct, how does the result of the fingerprint plot of correlates with the heatmap result? In the fingerprint we can see a stronger enrichment signal for G1E_Tal1_R1 as for G1E_Tal1_R2 but in the heatmap it is the otherway round. Furthermore, we can not see an enrichment for megakaryocytes_Tal1_R1 and R2 in the fingerprint plot, but a strong signal in the heatmap. How does this fit together?

The heatmap from the tutorial:

![image](https://user-images.githubusercontent.com/13435136/28918635-629475c4-784a-11e7-9bbd-4f90b4bb5aa4.png)

5. We would like to have the confirmation that the material is right as it is and that we are wrong. In the other case we would like to help to improve the material. 

Can you please do a review of the training material concerning our issues? @malloryfreeberg @bgruening @bebatut  @erxleben Thanks a lot!

@pavanvidem Would it be possible for you to proof read the material? Thanks a lot!

**Comment by bgruening** [comment]
ping @malloryfreeberg and @MoHeydarian 
Can you comment here?

**Comment by malloryfreeberg** [comment]
**Response to point 1:**

I re-downloaded the zenodo files and confirmed that the correct SRR identifiers are in all the reads. The regions we chose to select for the tutorial resulted in downsampling to ~10% of the full datasets for each library. The original "full" datasets themselves exhibited a range of sizes reflective of the ranges observed for the downsampled data. In each case, the downsampled zenodo file was ~10% of the full data file size. Original data from GEO: GSE51338.

```
G1E_Tal1_R1_downsampled_SRR492444.fastqsanger
Full file size: 378Mb
Downsample size: 32Mb (8.5% of full dataset)

G1E_Tal1_R2_downsampled_SRR492445.fastqsanger
Full file size: 551Mb
Downsample size: 63Mb (11.5%)

G1E_input_R1_downsampled_SRR507859.fastqsanger
Full file size: 404Mb
Downsample size: 36Mb (8.9%)

G1E_input_R2_downsampled_SRR507860.fastqsanger
Full file size:    4Gb
Downsample size: 433Mb (10.8%)


Megakaryocyte_input_R1_downsampled_SRR492453.fastqsanger
Full file size: 712Mb    
Downsample size: 59Mb (8.3%)

Megakaryocyte_input_R2_downsampled_SRR492454.fastqsanger
Full file size:    4Gb
Downsample size: 411Mb (10.3%)

Megakaryocyte_Tal1_R1_downsampled_SRR549006.fastqsanger
Full file size:    3Gb
Downsample size: 350Mb (11.7%)

Megakaryocytes_Tal1_R2_downsampled_SRR549007.fastqsanger
Full file size:    4Gb
Downsample size: 371Mb (9.3%)
```

The original "full" file sizes I copied directly from the corresponding entries in SRA, from which the original datasets were downloaded. For example, the [G1E input datasets](https://www.ncbi.nlm.nih.gov/sra/?term=SRR507859) had two replicates published on different dates, with one replicate (R1, SRR507859) size 404.1Mb and the second replicate (R2, SRR507860) size 4.6Gb, more than ten times larger as R1. This explains the 10-fold difference in the downsampled files sizes for G1E input R1/R2.

@MoHeydarian has the Galaxy workflows that actually did the downsampling, so he can confirm whether the samples were swapped or not.

**Response to points 2 & 4:**

I am not intimately familiar with the `deepTools` tools, other than using them to generate this tutorial. However, I have definitely seen non-typical results from running other tutorials on downsampled data. For example, running differential gene expression on a downsampled set of RNA-seq data (where typically reads corresponding to interesting differentially expressed transcripts are kept) results in violation of some DE tools' assumption that *most transcripts/genes are not differentially expressed*. So, perhaps the explanation for some of the unexpected `deepTools` outputs can be explained by the fact that the tutorial uses smaller - and thus potentially biased - ChIP-seq datasets. This inconsistency has always been a discussion point with creating tutorials based on downsampled data: the trade-off between execution time and representation of a "real" experiment (i.e. a full dataset).

So, while I can't speak directly to the inconsistencies between the `plotFingerprint` and `plotHeatmap` tools (perhaps someone with more experience running `deepTools` tools can), I suspect some of the non-typical behavior is caused by the downsampled nature of the tutorial data.

 I hope these responses begin to address some of the issues with the Tal1 ChIP-seq tutorial.
